src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -124,11 +124,14 @@ private CommandResult executeEditTutor(Model model) throws CommandException {
         Tutor tutorToEdit = lastShownList.get(index.getZeroBased());
         Tutor editedTutor = createEditedTutor(tutorToEdit, (EditTutorDescriptor) editPersonDescriptor);
 
+
         if (tutorToEdit.equals(editedTutor)) {
             throw new CommandException(MESSAGE_UNCHANGED_TUTOR);
         }
-        if (!tutorToEdit.isSamePerson(editedTutor) && model.hasTutor(editedTutor)) {
-            throw new CommandException(MESSAGE_DUPLICATE_TUTOR);
+
+        if (!tutorToEdit.getPhone().equals(editedTutor.getPhone())
```

Comment:
Maybe you want to consider adding a comment here since this is quite difficult to understand 😅